lib/commands/show.js

```diff
@@ -152,14 +152,20 @@ function showProblem(problem, argv) {
   log.printf('* %s', problem.category);
   log.printf('* %s (%s%%)', h.prettyLevel(problem.level), problem.percent.toFixed(2));
 
-  if (filename)
-    log.printf('* Source Code:       %s', chalk.yellow.underline(filename));
+  if (problem.likes)
+    log.printf('* Likes:    %s', problem.likes);
+  if (problem.dislikes)
+    log.printf('* Dislikes: %s', problem.dislikes);
```

Comment:
Do we need to align the whitespaces here?

Reply:
Since the console output is only needed for data extraction, whitespace alignment is not needed...I just wrote it this way to be in consistency with the old `leetcode-cli` behavior.